ghost/core/core/server/data/migrations/versions/5.21/2022-10-25-12-05-backfill-missed-products-columns.js

```diff
@@ -0,0 +1,35 @@
+const logging = require('@tryghost/logging');
+const {createTransactionalMigration} = require('../../utils');
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        logging.info(`Fixing currency/monthly_price/yearly_price values for default paid tiers`);
+
+        const currencyUpdated = await knex('products')
+            .update('currency', 'usd')
```

Comment:
@allouis I've inserted it as lowercase here as that's what we seem to insert in existing entries, but your original SQL was uppercase. Does it matter? I presume we handle the lowercase/uppercase appropriately

Reply:
It's all good - the code handles converting them 👍